src/main/scala/jawnfs2/package.scala

```diff
@@ -21,7 +19,7 @@ package object jawnfs2 {
   def parseJson[F[_], A, J](mode: AsyncParser.Mode)(implicit F: ApplicativeError[F, Throwable], A: Absorbable[A], facade: RawFacade[J]): Pipe[F, A, J] = {
     def go(parser: AsyncParser[J])(s: Stream[F, A]): Pull[F, J, Unit] = {
       def handle(attempt: Either[ParseException, collection.Seq[J]]) =
-        attempt.fold(Pull.raiseError[F], js => Pull.output(Chunk.seq(js)))
+        attempt.fold(Pull.raiseError[F], js => Pull.output(Chunk.seq(new SeqWrapper(js))))
```

Comment:
fs2 didn't qualify `Chunk.seq`'s argument, so it's `collection.Seq` (matching jawn) until Scala 2.13 and `collection.immutable.Seq` starting in Scala 2.13.

This wraps it unconditionally, though we might consider Scala version folders to avoid the wrapper until the it's needed.  Or maybe we could plead with the fs2 team to create a chunk constructor that takes a `collection.Seq` and let them optimize it.

We get no guarantee from jawn, but we're dealing with an `ArrayBuffer`, and could also consider a pattern match here that calls `Chunk.buffer` on the happy path.